src/Footer/Legal/Legal.jsx

```diff
@@ -1,84 +1,111 @@
 import styles from "./Legal.module.scss";
 import TrackedLink from "../../TrackedLink";
 import { footerClickEventAction } from "../../tracker";
+import { Grid, GridItem } from "@nice-digital/nds-grid";
+
+const CERTIFICATE = {
```

Comment:
Pls we update var name so its clear which certificate this is

Also doesn't need to be upper case